src/containers/__Routing/Routing.js

```diff
@@ -67,12 +68,22 @@ const RoutingBody = (props) => {
         }
     };
 
+    const gestureDemoPageCommand = {
+        command: ["Mirror mirror on the wall Go to gesture demo page"],
+        answer: "Going to gesture demo",
+        func: () => {
+            loggingContext.addLog("Voice Command: Going to gesture demo");
```

Comment:
I'm curious how well the voice recognition algorithm interprets the word "gesture". I haven't tested it out yet, does it seem to be working for you?

Reply:
Looks good.  Before this gets merged, can you add a "Mirror Mirror got to gesture demo page" command, Here is the change.
command: ["Mirror mirror on the wall Go to gesture demo page", "Mirror mirror Go to gesture demo page"],

Reply:
It seems to read "gesture" ok for me as long as I don't mumble it, we may have to change it so something else if it won't work flawlessly.  Is there a reason why many of our comments have the word "Go"  capitalized? is that something that enhances the readability of the "go"? I have made the changes that you guys suggested to me, I will push my current code up.

Reply:
The useSpeechRecogition hook converts all these strings to lower case letters so it doesn't matter that it is capitalized. Great observation though! That is something we should probably standardize.